jest.init.js

```diff
@@ -6,7 +6,22 @@
 
 import { config } from "@vue/test-utils";
 import InlineSvg from "vue-inline-svg";
+import Toast from "vue-toastification";
 
-config.global.components = {
-  InlineSvg,
+import mixpanel from "mixpanel-browser";
+mixpanel.init(process.env.VUE_APP_MIXPANEL_PROJECT_TOKEN);
```

Comment:
if you are using environment variables - you need to set them in `.github/workflows/ci.yml` but my understanding is that this is just a mock and hence, we don't need to initialize with our token and hence, this line can be removed/